newton/examples/robot/example_robot_humanoid.py

```
@@ -113,7 +115,18 @@ def render(self):
         self.viewer.end_frame()
 
     def test(self):
-        pass
+        newton.examples.test_body_state(
+            self.model,
+            self.state_0,
+            "all bodies are above the ground",
+            lambda q, qd: q[2] > 0.01,
```

Ruff (0.13.1)

Unused lambda argument: `qd`

(ARG005)